frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -123,25 +123,79 @@ const HostSummary = ({
   );
 
   const renderDiskSpace = () => {
+    const diskSpaceTooltip = () => {
+      const diskSpaceAvailable = titleData.gigs_disk_space_available;
+      switch (true) {
+        case diskSpaceAvailable < 16:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Not enough disk space <br />
+              available to install most <br />
+              small operating systems <br />
+              updates.
+            </span>
+          );
+        case diskSpaceAvailable < 32:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Not enough disk space <br />
+              available to install most <br />
+              large operating systems <br />
+              updates.
+            </span>
+          );
+        default:
+          return (
+            <span className={`${baseClass}__tooltip-text`}>
+              Enough disk space available <br />
+              to install most operating <br />
+              systems updates.
+            </span>
+          );
+      }
```

Comment:
This switch statement and associated tooltip text is duplicated between here and `HostTableConfig.tsx`. Could it be abstracted to a single instance and imported?

Having the two implementation on two separate issues is kind of misleading. I should have combined them into a single issue since the implementation is identical.

Reply:
Totally agree, I'll rebase and refactor on this PR